src/ripple/core/JobQueue.h

```diff
@@ -27,6 +27,8 @@
 #include <ripple/core/Stoppable.h>
 #include <ripple/core/impl/Workers.h>
 #include <ripple/json/json_value.h>
+#include <boost/range/begin.hpp>
+#include <boost/range/end.hpp>
```

Comment:
@HowardHinnant I believe you that we need these, I just don't know why. I'm curious what's going on here.

Reply:
It's a regression bug in boost.  Here's the boost bug report:  https://github.com/boostorg/coroutine/issues/46

This is nothing but a cheap workaround for rippled to keep us forwards and backwards compatible with boost versions.